apps/backend/src/modules/tasks/task.repository.ts

```diff
@@ -0,0 +1,58 @@
+import { RelationName } from "~/libs/enums/relation-name.enum.js";
+import { type Repository } from "~/libs/types/types.js";
+import { TaskEntity } from "~/modules/tasks/task.entity.js";
+
+import { TaskStatus } from "./libs/enums/enums.js";
+import { type TaskModel } from "./task.model.js";
+
+class TaskRepository implements Repository {
+	private taskModel: typeof TaskModel;
+
+	public constructor(taskModel: typeof TaskModel) {
+		this.taskModel = taskModel;
+	}
+
+	create(): Promise<unknown> {
+		return Promise.resolve();
+	}
+
+	public delete(): ReturnType<Repository["delete"]> {
+		return Promise.resolve(true);
+	}
+
+	find(): Promise<unknown> {
```

Comment:
For both task service and task repository:
B30. All class properties and methods are marked with member access (private , public or protected).